server/container_create_linux.go

```diff
@@ -1254,7 +1165,7 @@ func setupSystemd(mounts []rspec.Mount, g generate.Generator) {
 		g.AddMount(tmpfsMnt)
 	}
 
-	if cgroups.IsCgroup2UnifiedMode() {
+	if node.CgroupIsV2() {
```

Comment:
OK I just saw there is a version of the function the way you want it, without an error returned as a second parameter. Perhaps we can convert everyone to just use it?

Or, if you don't like the name, you can do
```go
package node
...
import (
...
"github.com/opencontainers/runc/libcontainer/cgroups"
...

var CgroupIsV2 = cgroups.IsCgroup2UnifiedMode()
```
and use `node.CgroupIsV2()` the way you do.